pkg/render/logstorage.go

```diff
@@ -1476,13 +1459,13 @@ func (es elasticsearchComponent) oidcUserRole() []client.Object {
 					APIGroups:     []string{""},
 					Resources:     []string{"configmaps"},
 					ResourceNames: []string{OIDCUsersConfigMapName},
-					Verbs:         []string{"get", "list", "watch"},
+					Verbs:         []string{"*"},
```

Comment:
Why are we increasing access to all verbs if we are no longer watching the config map ?

Reply:
We are moving the code to manage this ConfigMap and Secret over to kube-controller, I've added updated this PRs description and added more details in the kube-controller PR.

Reply:
I see we have create, delete, get & watch. Sounds good.